src/pynwb/ophys.py

```diff
@@ -271,6 +271,27 @@ def add_roi(self, **kwargs):
             rkwargs['voxel_mask'] = voxel_mask
         return super(PlaneSegmentation, self).add_row(**rkwargs)
 
+    def pixel_to_image(self, pixel_mask, width, height):
+        image_matrix = np.zeros((width, height))
+        for mask in pixel_mask:
```

Comment:
I think something like this might be a bit faster than setting all the elements one-by-one in the for loop:

```
npmask = np.asarray(pixel_mask)
image_matrix[npmask[:,0], npmask[:,1]] =  npmask[:,-1]
return image_matrix
```